### ISSUE TITLE
Make fonts on the blog match changes in the web app

### ISSUE THREAD

**Issue by joshsmith** [code-corps/blog.codecorps.org-site#10]
We changed the fonts in the web app to remove the FOUT and clean things up. We should do the same over here.

**Comment by riskiii** [comment]
Should I use https://github.com/code-corps/code-corps-ember/blob/develop/app/styles/_fonts.scss as the base for it?

**Comment by joshsmith** [comment]
@riskiii yes, sorry for delay here! We need to bring the fonts in line with that.

**Comment by joshsmith** [comment]
And let me know if there are any places that aren't clearly covered by that.
